pkg/store/store.go

```diff
@@ -21,12 +21,12 @@ var (
 	signaturePrefix = "c"
 	statePrefix     = "s"
 	metaPrefix      = "m"
+	heightPrefix    = "0"
```

Comment:
mainKV uses `0` prefix: https://github.com/rollkit/rollkit/blob/main/node/full.go#L48
using something else for heightPrefix better?